### ISSUE TITLE
Source tests failing in appveyor

### ISSUE THREAD

**Issue by phlptp** [GMLC-TDC/HELICS-src#290]
The test for source app are failing regularly in appveyor.   They have been disabled by default for the moment, but the source of the failure needs to be determined and fixed.

**Comment by phlptp** [comment]
I am still not sure what the root cause of this is so I need to postpone it to 1.3

**Comment by phlptp** [comment]
looks like there may be timeout cause for the application_api test but the source tests are no longer failing in the issue_363 branch.  So once that is merged I will close this issue.
